website/versioned_docs/version-V1/sdk-reference/react.mdx

```diff
@@ -226,6 +228,10 @@ If you specify an allowed type but it mismatches the setting kind, an error mess
 
 ## Anatomy of `useConfigCatClient()`
 
+| Parameters     | Description                                                                                                                |
+| -------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `providerId`   | Optional, Identifier of `ConfigCatProvider`. [More about multiple providers](#using-multiple-providers).                   |
```

Comment:
```suggestion
| `providerId`   | Optional. Instructs the ConfigCat Client to use a specific `ConfigCatProvider` instead of the default one. Use it if you have multiple `ConfigCatProvider`s in your code. [More about multiple providers](#using-multiple-providers).                   |
```